server/src/main/java/org/solarbank/server/dto/UserInputDto.java

```diff
@@ -0,0 +1,24 @@
+package org.solarbank.server.dto;
+
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.DecimalMax;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
+import lombok.Data;
+
+@Data
```

Comment:
So under the hood `Spring` uses `jackson` for serialising java to json and deserialising json to java fields. By default it requires that the json properties match exactly with the names of the fields in the class.

In your code your using `camelCase` by convention, but json normally uses `PascalCase`. So we need to specify the strategy for `jackson` to use to convert from pascal to camel case, when it is deserialising the json. e.g.

`@JsonNaming(PropertyNamingStrategies.UpperCamelCaseStrategy.class)`

Reply:
added annotation for deserialising JSON